src/interfaces/category.ts

```diff
@@ -0,0 +1,7 @@
+export interface CategoryInterface {
```

Comment:
We already know that it is an interface according top key-word interface in the beginning
```suggestion
export interface Category {
```